src/Core/src/Platform/Android/PickerManager.cs

```diff
@@ -59,8 +60,10 @@ public static Java.Lang.ICharSequence GetTitle(Color titleColor, string title)
 			if (titleColor == null)
 				return new Java.Lang.String(title);
 
-			var spannableTitle = new SpannableString(title ?? string.Empty);
-			spannableTitle.SetSpan(new ForegroundColorSpan(titleColor.ToPlatform()), 0, spannableTitle.Length(), SpanTypes.ExclusiveExclusive);
```

Comment:
`SpannableString.Length()` is supported on android 29.0+

Reply:
Is this according to Google docs or our annotations? This seemed suspicious and I just tested on a 23 emulator and it works. I wonder if there is just an issue in the binding?

Reply:
> Is this according to Google docs or our annotations?

Annotations 
<img width="236" alt="image" src="https://user-images.githubusercontent.com/45579687/165596337-b9073b49-ba4a-4ec6-aece-8333c603b7bc.png">
You mean the annotation might incorrect?

Reply:
https://github.com/xamarin/xamarin-android/issues/6962#issuecomment-1111379269